libres/lib/job_queue/job_node.cpp

```diff
@@ -594,7 +593,7 @@ bool job_queue_node_update_status_simple(job_queue_node_type *node,
         job_queue_node_set_status(node, new_status);
     }
     pthread_mutex_unlock(&node->data_mutex);
-    return status_change;
+    return;
```

Comment:
Is this `return` statement still required?

Reply:
ah thanks, very good point, no, it is not : )

Reply:
just updated the branch (force-pushed the change of removing that line)